CHANGELOG

```diff
@@ -2,6 +2,11 @@ forthcoming
 -----------------------------------
 - []
 
+
+1.2.3
```

Comment:
It's probably better to call it `next` or `forthcoming` or such rather than mentioning yet unreleased version. After all, it might be a minor version bump.